src/entity/User.ts

```diff
@@ -118,12 +126,13 @@ export type AddUserData = Pick<
   | 'createdAt'
   | 'github'
   | 'twitter'
-  | 'referral'
+  | 'referralId'
   | 'infoConfirmed'
   | 'profileConfirmed'
   | 'acceptedMarketing'
   | 'timezone'
 >;
+export type AddUserDataPost = { referral: string } & AddUserData;
```

Comment:
`{ referral: string }` this part is no longer applicable right?

Reply:
Yep, it's just added there to retroactively support existing implementations.
We merge referral to referralId if it exists.

Reply:
It's a new type only for the post option